sharding-core/src/main/java/io/shardingsphere/core/executor/ShardingExecuteEngine.java

```diff
@@ -53,6 +53,15 @@ public ShardingExecuteEngine(final int executorSize) {
         MoreExecutors.addDelayedShutdownHook(executorService, 60, TimeUnit.SECONDS);
     }
     
+    /**
```

Comment:
it seem that we should extract ListeningExecutorService to a Util class for common use, as CommandExecutorContext will not use other feature in ShardingExecuteEngine.